foxsi.pro

```diff
@@ -0,0 +1,42 @@
+; June 2016
+; STARTUP FOXSI
+; Description: 
+;
+;
+
+PRO FOXSI, YEAR
+
+COMMON FOXSI
```

Comment:
FYI, I've gotten some feedback that people really don't like our use of common. Not sure if there is any way around it though.